middleware.js

```diff
@@ -165,6 +165,27 @@ async function requireSessionUser (req, res, next) {
   next()
 }
 
+async function requireAdminUser (req, res, next) {
+  const user = await _getRequestSessionUser(req)
+  if (!user) {
+    const queryParams = new URLSearchParams(req.query).toString()
+    return res.redirect(`/oauth/init?${queryParams}`)
+  }
+  const fxaProfileData = await FXA.getProfileData(user.fxa_access_token)
+  const admins = AppConstants.ADMINS?.split(',')
+  const isAdmin = admins.includes(JSON.parse(fxaProfileData).email)
+
+  if (!isAdmin || (fxaProfileData.hasOwnProperty('name') && fxaProfileData.name === 'HTTPError')) {
+    delete req.session.user
+    return res.sendStatus(401)
+  }
```

Comment:
I think if they are not an admin user, we should return a 401 but leave the user logged in. We can also address the [`no-prototype-builtins` eslint warning](https://eslint.org/docs/latest/rules/no-prototype-builtins)
```suggestion
  const hasFxaError = Object.prototype.hasOwnProperty.call(fxaProfileData, "name") && fxaProfileData.name
  if (hasFxaError) {
    delete req.session.user
  }
  if (!isAdmin || hasFxaError) {
    return res.sendStatus(401)
  }
```